internal/benthos/benthos-builder/builders/sql.go

```diff
@@ -374,16 +374,23 @@ func (b *sqlSyncBuilder) BuildDestinationConfig(ctx context.Context, params *bb_
 			}
 		}
 
-		columnTypes := []string{}
+		columnTypes := []string{} // use map going forward
```

Comment:
what does this comment mean?

Reply:
I don't want to continue using column types and column default properties as lists. I want them to be maps where key is column name because I have to do all this extra work to keep them all in the same order as the columns list